scripts/dist-tag.ts

```diff
@@ -0,0 +1,51 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+// tslint:disable: no-implicit-dependencies
+
+import { logging } from '@angular-devkit/core';
+import { execSync } from 'child_process';
+import { packages, stableToExperimentalVersion } from '../lib/packages';
+
+interface DistTagOptions {
+  /**
+   * Version must be specified in format d+.d+.d+ where d is a 0-9 digit.
+   * This must be a stable version with major version > 0.
+   * The script will automatically convert stable version to experimental.
+   */
+  version: string;
+  /**
+   * Tag is usually "latest" or "next", but could also be "v10-lts" for example.
+   */
+  tag: string;
+}
+
+/**
+ * This function adds a tag to all public packages in the CLI repo.
+ */
+export default function(args: DistTagOptions, logger: logging.Logger) {
+  const {version, tag} = args;
+  if (!version || version.startsWith('v')) {
+    throw new Error('Version must be specified in format d+.d+.d+');
+  }
+  if (version.startsWith('0')) {
+    throw new Error('Version must be "stable", with major version > 0');
```

Comment:
If I saw this error message I'm not sure I'd understand it. My interpretation looking the code is that this is trying to disallow `0.x.x` packages. Maybe we can state this a little more clearly? The definition of "stable" is a bit vague.

The doc comment for `version` is also a bit unclear because of its use of "stable".

Reply:
Removed all mentions of "stable", and provided suggestion instead.